Mathlib/Data/Matrix/Composition.lean

```diff
@@ -0,0 +1,132 @@
+/-
+Copyright (c) 2024 Yunzhou Xie. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kevin Buzzard, Yunzhou Xie
+-/
+import Mathlib.Data.Matrix.Basic
+
+/-!
+# Composition of matrices
+This file shows that Mₙ(Mₘ(R)) ≃ Mₙₘ(R), Mₙ(Mₘ(R)) ≃ Mₘ(Mₙ(R)), Mn(Rᵒᵖ) ≃ₐ[K] Mₙ(R)ᵒᵖ
+and also different levels of equivalence when R is an AddCommMonoid,
```

Comment:
Can you include a new section like this
```
## Main results
```

Reply:
done

Reply:
Sorry, I haven’t made it clear. Module docs string is using markdown syntax. We typically include a section dedicated to list important results in the file.